.github/workflows/promote.yaml

```diff
@@ -22,11 +22,20 @@ jobs:
 
       - name: Re-tag and promote awx-operator image
         run: |
+          # Promote operator image
           docker pull ghcr.io/${{ github.repository }}:${{ github.event.release.tag_name }}
           docker tag ghcr.io/${{ github.repository }}:${{ github.event.release.tag_name }} quay.io/${{ github.repository }}:${{ github.event.release.tag_name }}
           docker tag ghcr.io/${{ github.repository }}:${{ github.event.release.tag_name }} quay.io/${{ github.repository }}:latest
           docker push quay.io/${{ github.repository }}:${{ github.event.release.tag_name }}
           docker push quay.io/${{ github.repository }}:latest
+          # Promote bundle image
+          docker pull ghcr.io/${{ github.repository }}-bundle:v${{ github.event.release.tag_name }}
+          docker tag ghcr.io/${{ github.repository }}-bundle:v${{ github.event.release.tag_name }} quay.io/${{ github.repository }}-bundle:v${{ github.event.release.tag_name }}
```

Comment:
This PR looks good, CI is just complaining because some lines are too long.  You should be able to just add `#noqa` at the end of those lines.